api/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsAuthenticated(BasePermission):
+    """
+    Allows access only to authenticated users.
+    """
+
+    def has_permission(self, request, view):
+        return request.auth
```

Comment:
You mean for example request.auth is not none? request.auth is not a boolean and the naming suggests the return type should be a boolean